src/main/java/marquez/api/models/DatasourcesResponse.java

```diff
@@ -0,0 +1,19 @@
+package marquez.api.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+@ToString
```

Comment:
Why not just use `@Data`?

Reply:
You can follow the recommended approach for writing response payloads in #355

Reply:
Please see PRs #394, #387, and #384, where this is fixed.